### DISCUSSION TITLE
Helm Chart - v1.10.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [Azure/api-management-self-hosted-gateway#285] [Announcements]
[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/azure-api-management)](https://artifacthub.io/packages/helm/azure-api-management/azure-api-management-gateway/1.10.0)

*Chart: v1.10.0 | App: v2.6.0*

### Features

- Use container image v2.6.0 ([release notes](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/Container-v2.6.0))
- Automatically configure socket volume when using syslog (https://github.com/Azure/api-management-self-hosted-gateway/pull/283)
- Introduce startup probe (https://github.com/Azure/api-management-self-hosted-gateway/pull/281 - Contributed by @RomanDvorsky 🙏 )
- Introduce capability to dynamically allocate new node ports for load balancer service (https://github.com/Azure/api-management-self-hosted-gateway/pull/280 - Contributed by @allevimi 🙏)

### Fixes / Changes

- Use high-availability by default (https://github.com/Azure/api-management-self-hosted-gateway/pull/275)
- Fix a bug where assigning 0 replicas was not applied (https://github.com/Azure/api-management-self-hosted-gateway/pull/277 - Contributed by @RomanDvorsky 🙏)

### Breaking Changes

None.

### Removal

None.

## New Contributors

- @allevimi

**Full Changelog**: https://github.com/Azure/api-management-self-hosted-gateway/compare/v1.9.0...v1.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/v1.10.0'>Helm Chart - v1.10.0</a>.</em>

**Comment by RomanDvorsky** [comment]
Cool! Thx for noticing!

Regards,
R.

From: Tom Kerkhove ***@***.***>
Sent: Tuesday, May 21, 2024 6:01 PM
To: Azure/api-management-self-hosted-gateway ***@***.***>
Cc: Dvorsky, Roman [External] ***@***.***>; Mention ***@***.***>
Subject: [Azure/api-management-self-hosted-gateway] Helm Chart - v1.10.0 (Discussion #285)


[Artifact Hub]<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6YzJiMjY0ND5lMjY+NWUzZjVhZTVmZTE+Mz4wZjdiM2E+NWI1NT9lMiFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fartifacthub.io%2fpackages%2fhelm%2fazure-api-management%2fazure-api-management-gateway%2f1.10.0>

Chart: v1.10.0 | App: v2.6.0

What is new?
Features

  *   Use container image v2.6.0 (release notes<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6Njc/YjczZDMyPmY0ZGZlP2EzYTI0NTA3NzdiYjUwZGYwNGFmN2YzZSFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2freleases%2ftag%2fContainer-v2.6.0>)
  *   Automatically configure socket volume when using syslog (#283<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6ZTM1P2Y1YzU/P2UyYjc+NjViPzIxMjMxNTc+PjE0MTcyZWFiYTIyYiFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2fpull%2f283>)
  *   Introduce startup probe (#281<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6M2YwYWJjMjZjPzdlMDc1MmI1ZmE3ZjA1YjQ2ZDc3YmVlZDZlZmViMyFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2fpull%2f281> - Contributed by @RomanDvorsky<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6ZWU3ZTA+MmIwYzNkM2Y/PzExMzEzYTU3P2IyYWMwNGZiMjJkMDU2MiFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fRomanDvorsky> 🙏 )
  *   Introduce capability to dynamically allocate new node ports for load balancer service (#280<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6ZTcyZGIzYWQzZTdiPmRlMGZmNGM0NDNkP2I+YzIxMTQ+Mj5hZD5lMSFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2fpull%2f280> - Contributed by @allevimi<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6YTVjMmRjNTMzYjc/YjNlYzA/NWRjMDA/YjYzZWU1NTVhYzI/MWMyYSFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fallevimi> 🙏)

Fixes / Changes

  *   Use high-availability by default (#275<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6YTUxNDI/M2I/YzI+YWVkMjBkNz41M2Q0YWI0Mj8/ZGVkYzRmZTM3MiFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2fpull%2f275>)
  *   Fix a bug where assigning 0 replicas was not applied (#277<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6ZjNmNjMyMzRhP2IxMmQyZmMyNWVhM2MwNzE2MjQ+MmZkZDQwZmYyZCFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2fpull%2f277> - Contributed by @RomanDvorsky<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6ZWU3ZTA+MmIwYzNkM2Y/PzExMzEzYTU3P2IyYWMwNGZiMjJkMDU2MiFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fRomanDvorsky> 🙏)

Breaking Changes

None.

Removal

None.

New Contributors

  *   @allevimi<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6YTVjMmRjNTMzYjc/YjNlYzA/NWRjMDA/YjYzZWU1NTVhYzI/MWMyYSFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fallevimi>

Full Changelog: v1.9.0...v1.10.0<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6YjRiNjY2ZDU1PjYzPjM2ZGZlMDFhMmUxPz4+ZTM+NWI+YzI/Pj4yMyFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2fcompare%2fv1.9.0...v1.10.0>

________________________________
This discussion was created from the release Helm Chart - v1.10.0<https://inbound-mx.oriflame.com/fmlurlsvc/?fewReq=:B:JVc9NjI2PSFxOjUpNyFuYzo3Nj03NiF0bmBpZnNydWI6MzI2MWY/MTM3M2RhP2U1NWQ1YzViYmYzYTYyZWM1MTJkMzVjYTFlPiFzOjYwNjE0NzA1MDQhdm5jOjMzS0A2RDRANzY+NTExKjMzS0A2RDRONzY+NTExIXVkd3M6dWNxaHV0bH5HaHVuYWtmamIpZGhqIWQ6MjUhb2NrOjc=&url=https%3a%2f%2fgithub.com%2fAzure%2fapi-management-self-hosted-gateway%2freleases%2ftag%2fv1.10.0>.
